packages/atoms/availabilitylist/AvailabilityList.tsx

```diff
@@ -0,0 +1,72 @@
+import { NewScheduleButton } from "availabilitylist/NewScheduleButton";
```

Comment:
Can we rename `atoms/availabilitylist/` to `atoms/availability-list/` ?

Reply:
Then, I would re-name `AvailabilityList.tsx` to `index.tsx` so that it's clear it's the main file within `availability-list`

Reply:
Then, I would create `atoms/availability-list/components` folder where to put `EmptyScreen.tsx`, `Form.tsx` etc. and if some of those need to be split into smaller parts to improve readability then create a child folder, for example, `atoms/availability-list/components/form` where `Form.tsx` then becomes `index.tsx`